api/external/events.html

```diff
@@ -0,0 +1,530 @@
+                                                </ul>
+                                            </td>
+                                        </tr>
+                                        <tr class="field-odd field">
+                                            <th class="field-name">Returns:</th>
+                                            <td class="field-body">Information for the updated member event type.</td>
+                                        </tr>
+                                        <tr class="field-even field">
+                                            <th class="field-name">Raises:</th>
+                                            <td class="field-body">
+                                                <ul style="margin: 0; padding: 0;">
+                                                    <li>
+                                                        <tt class="docutils literal">
+                                                            <span class="pre">Http404</span>
+                                                        </tt>
+                                                        if no member event type with the specified ID is found.
+                                                    </li>
+                                                    <li>
+                                                        <tt class="docutils literal">
+                                                            <span class="pre">Http400</span>
+                                                        </tt>
+                                                        if no display name and event type are provided.
+                                                    </li>
+                                                </ul>
+                                            </td>
+                                        </tr>
+                                    </tbody>
+                                </table>
+                                <p>
+                                    <div class="sample-response hide-response">
+                                        <span style="font-weight:bold;">Sample Response</span>
+                                        <a href="#" class="resp-trigger">[<span class="tshow">show</span><span class="thide">hide</span>]</a>
+<pre class="response">
+    <b>PUT /100/events/200</b>
+    {
+        "account_id": 100,
+        "created_at": "@D:2025-11-25T10:32:44",
+        "event_type": "Purchased Tickets",
+        "display_name": "purchased_tickets",
+        "event_type_id": 200
+    }
+</pre>
+                                    </div>
+                                </p>
+                            </dd>
+                        </dl>
+                        <dl class="delete">
+                            <dt id="delete--#account_id-member_event_type_by_id">
+                                <tt class="descname">Delete </tt>
```

Comment:
Let's use all caps to match the style within the other endpoints

```suggestion
                                <tt class="descname">DELETE </tt>
```